web/openapi/openapi.yaml

```diff
@@ -260,6 +296,11 @@ paths:
                         - id: 2
                           definition: publish /bar
                           instrumented: true
+                      diagnostics: []
+                      metrics:
+                        total:
+                          in: {}
+                          out: {}
```

Comment:
A real example would be very useful here.

Reply:
I agree, but can you add the example to where you define `#/components/schemas/Metrics`? You could then consider removing the whole `example:` section of `/pipeline/list`, as this should automatically use the examples provided by the schema.

Reply:
I added examples to the Metrics schema and removed the `/pipeline/list` examples in https://github.com/tenzir/tenzir/pull/3376/commits/324fa9d2399798c712324aa053732e7ea3a03f3e